pkg/controller/provider/model/vsphere/model.go

```diff
@@ -264,6 +264,7 @@ type VM struct {
 	Devices               []Device       `sql:""`
 	NICs                  []NIC          `sql:""`
 	Disks                 []Disk         `sql:""`
+	Controllers           []Controller   `sql:""`
```

Comment:
doe it make sense to do the controllers a map by key, instead of an array ?

Reply:
I wanted to go that way but at the same time, I did not want to break the convention.
I think most of these could be maps

Reply:
i am ok with both ways ...